docs/api.md

```diff
@@ -1535,6 +1536,27 @@ const iPhone = puppeteer.devices['iPhone 6'];
 
 List of all available devices is available in the source code: [src/common/DeviceDescriptors.ts](https://github.com/puppeteer/puppeteer/blob/main/src/common/DeviceDescriptors.ts).
 
+#### page.emulateCPUThrottling(rate)
+
+- `rate` <?[number]> Rate at which the CPU will be throttled (2x, 3x, ...). Passing `null` disables cpu throttling.
```

Comment:
Based on this description, would `factor` be a clearer name?

Reply:
I don't have strong feelings about the naming. `rate` is used in the CDP definition - that's why I chose it here.
In Chromium, this gets converted into a percentage (2 -> 200%, 2.5 -> 250%) so we could also change the argument to be a percentage instead. LMKWYT!

Reply:
I don't feel strongly either but I do think that `factor` is clearer/more self-documenting. Let's not do the percentages for now — it seems confusing IMHO.

Reply:
I renamed it to factor for now :)